go-selinux/label/label_selinux.go

```diff
@@ -52,11 +53,11 @@ func InitLabels(options []string) (plabel string, mlabel string, Err error) {
 				return "", mountLabel, nil
 			}
 			if i := strings.Index(opt, ":"); i == -1 {
-				return "", "", fmt.Errorf("Bad label option %q, valid options 'disable' or \n'user, role, level, type, filetype' followed by ':' and a value", opt)
+				return "", "", errors.Errorf("Bad label option %q, valid options 'disable' or \n'user, role, level, type, filetype' followed by ':' and a value", opt)
```

Comment:
Wondering; should this also check if there's anything _after_ the colon? (e.g., `disable:` is currently "valid")

Reply:
Yeah, this is not a very good check; should be addressed separately

Reply:
@kolyshkin Do you approve of this patch?